### ISSUE TITLE
breaking change in patch release

### ISSUE THREAD

**Issue by zolex** [veracode/veracode-uploadandscan-action#21]
you just changed the deleteIncompleteScan from boolean to integer which is a breaking change, but you only bumped the patch version. the new release should have been 1.0.0. (or 0.3.0 if you had the possibility to translate false to 0 and true to 1 internally)

please consider SemVer for future versions, this release broke all our workflows.

**Comment by zolex** [comment]
now apart from the random version number bump, the feature seems not to work properly for us. see these logs:

![image](https://user-images.githubusercontent.com/398739/167557404-e316f6db-bfd9-4009-b275-b6bebe43c6fc.png)

**Comment by tjarrettveracode** [comment]
Hi folks, thanks for the feedback on this. We are taking the feedback on versioning to heart.

We have made an update to the underlying Java wrapper that this action is based on to introduce backward compatibility for the `deleteIncompleteScan` parameter. This does not unfortunately help folks who have already had to fix their pipelines but it does avoid the problem for users that have not yet run into the issue.

We have also realized that the action's behavior of always pulling the latest wrapper may not be desirable for cases like this where breaking changes are introduced. I have filed #22 to address this issue more explicitly.
